### PR TITLE
Use dynamic reconfigure for twist controller gain

### PR DESCRIPTION
**Pull request by fmauch**
as per subject

### CODE REVIEW

File: twist_controller/src/twist_controller.cpp [hunk 1]
```diff
@@ -51,6 +52,11 @@ bool TwistController::init(TwistCommandInterface* hw, ros::NodeHandle& n)
     return false;
   }
 
+  gain_ = n.param("twist_gain", 0.1);
+  server_.reset(new dynamic_reconfigure::Server<twist_controller::TwistControllerConfig>(n));
+  server_->setCallback(
+      boost::bind(&TwistController::reconfigureCallback, this, _1, _2));
```
Comment (stefanscherzinger) [inline twist_controller/src/twist_controller.cpp:58]: No std::bind available?

Comment (stefanscherzinger) [reply twist_controller/src/twist_controller.cpp:58]: @fmauch

**Review by stefanscherzinger — APPROVED** [review]
Nice, I like it.

File: twist_controller/src/twist_controller.cpp (hunk 1 above)
Comment (fmauch) [reply twist_controller/src/twist_controller.cpp:58]: Nope, it expects a `boost::function` object, see https://github.com/ros/dynamic_reconfigure/blob/7a3158394005c1650bf541198410a1308b628651/include/dynamic_reconfigure/server.h#L83-L85

Comment (stefanscherzinger) [reply twist_controller/src/twist_controller.cpp:58]: ok.

**Pull request merged**